File: AnimeQuiz/app/src/main/java/aik3/game/animequiz/MainActivity.java
package aik3.game.animequiz;

import androidx.appcompat.app.AppCompatActivity;
import androidx.navigation.NavController;
import androidx.navigation.fragment.NavHostFragment;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    private NavHostFragment host;
    private NavController controller;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        host = (NavHostFragment) getSupportFragmentManager().findFragmentById(R.id.nav_host_fragment);
        controller = host.getNavController();
        controller.navigate(R.id.mainFragment);
    }

    @Override
    protected void onPause() {
        super.onPause();
        host = (NavHostFragment) getSupportFragmentManager().findFragmentById(R.id.nav_host_fragment);
        controller = host.getNavController();
        controller.navigate(R.id.mainFragment);
    }
}

File: AQK/app/src/main/java/aik3/game/animequiz/MainActivity.kt
package aik3.game.animequiz

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.navigation.NavController
import androidx.navigation.fragment.NavHostFragment

class MainActivity : AppCompatActivity() {

    private lateinit var host: NavHostFragment;
    private lateinit var controller: NavController;

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        host = supportFragmentManager.findFragmentById(R.id.nav_host_fragment) as NavHostFragment
        controller = host.navController
        controller.navigate(R.id.mainFragment)
    }

    override fun onPause() {
        super.onPause()
        controller.navigate(R.id.mainFragment)
    }
}